receiver/azuremonitorreceiver/scraper.go

```diff
@@ -231,7 +247,17 @@ func (s *azureScraper) getResources(ctx context.Context) {
 				}
 				if resource.Location != nil {
 					attributes[attributeLocation] = resource.Location
+					attributes["cloud.region"] = resource.Location
```

Comment:
After stepping away I realized that setting these attributes should have been conditioned on whether these attributes were enabled in settings. I'm addressing that now with some light refactoring.

Reply:
the generated code will filter out the resource attributes based on what is enabled, so you should be good.